portality/api/v1/crud/applications.py

```diff
@@ -109,7 +110,7 @@ def update(cls, id, data, account):
 
         # finally save the new application, and return to the caller
         new_ap.save()
-        return ap
+        return new_ap
```

Comment:
Nice catch in applications. Does a test need to be added or change as well so that we make sure we return the new application to the user rather than the old object? I'm not saying you should add it in this PR, just if we should make a note to.